lib/psulib_blacklight/solr_manager.rb

```diff
@@ -59,8 +59,13 @@ def initialize_collection
       if next_collection_version == 1
         puts "creating collection #{next_collection_version}"
         create_collection
+        @update_config ||= modify_collection
+        create_alias
       else
         puts 'collection exists skipping.'
+        @update_config ||= modify_collection
+        # Return nil to indicate we didn't create a collection
+        nil
```

Comment:
It doesn't look like you need to rely on nil here. move the puts to the end.

Reply:
nil was put there specifically to allow the tests to pass, there's a test that expects nil, and couldn't determine what to put in there. _shrug_ didn't think it was such a big deal, sorry

Reply:
Explicitly returning nil can sometimes mean something's not quite right. Maybe the tests just need to be updated. I only see the method getting invoked from a rake task, so it would make sense if it was just:

``` ruby
else
  @update_config ||= modify_collection
  puts 'collection exists skipping.'
end
```

Reply:
gotcha, could have sworn this failed before, but could have been some of the other changes i needed to make, too.